Library/Formula/metashell.rb

```diff
@@ -10,20 +11,28 @@ class Metashell < Formula
     sha1 "76874d93ced1815ccfcbddb6da75e8c56aff44e4" => :mountain_lion
   end
 
-  depends_on "cmake" => :build
-
   needs :cxx11
 
+  # This patch is required because Mountain Lion uses an old compiler which breaks
```

Comment:
Has this been submitted anywhere? If so, please include in a link in the formula, Thanks!